anti.md

```diff
@@ -7,45 +8,113 @@ good way to meet their needs.
 
 ### Why you?
-_What excites you about this idea? How did you come up with it?_
 
+As a Harvey Mudd student, I have a strong interest in a more effective
+registration system. I've also spent some time working on Casey Chu's scheduler
+application ([here](https://github.com/bitsofpancake/hmc-scheduler)), so I'm a
+bit familiar with the systems involved.
+
+I came up with this idea by asking students what made them feel the worst when
+they were interacting with computers, and someone brought up portal. Between
+that and the general complaints about Portal I feel confident this would be
+helping to solve a genuine problem.
 
 ### Domain
-_Describe the project's domain in five words._
 
+Course Search and Registration
 
 ### Interface (syntax)
-_How might the user interact with the language? What does programming look 
-like? Why is this the right way to interact with the problem domain?_ 
 
+One could imagine that the language syntax would be similar to that of SQL for
+executing searches, something like:
+
+```
+> find CS, taught by Ran, number > 100, when I'm free - sort by number
+```
+
+One would also be able to register for classes:
+```
+> register CS105-1
+Registeration for [CS105-1 "Computer System" taught by Geoff Keuning] was
+successful!
+> register CS105
+Ambiguous command. Do you want to register for
+0. No class
+1. [CS105-1 "Computer System" taught by Geoff Kuening]
+2. [CS105-2 "Computer System" taught by Z Sweedyk]
+Enter Number > ...
+```
+
+Students should also be able to get info on their current schedule and more
+detailed information about each class.
+
+One could also set this up as a graphical interface, but the key thing is
+that the interface should be flexible and should have a single entry point.
+
+Student want to be able to alternate between dramatically different searches
+and even different actions (course details, registration, schedule details)
+without having to click through a bunch of buttons and screens.
 
 ### Operation (semantics)
-_What might happen when a program runs? How does a program interact with the
-user? What kinds of errors might occur, and how might they be communicated to
-the user?_
 
+When a search is run the system would have to hit the back-end registration
+system and perform the requested search. If something about the search could
+not be understood, that would be articulated to the user instead of trying to
+perform it.
+
+When a register command goes out the program would have to register the student
+for that class.
 
 ### Expressiveness
-_What should be easy to do in this language? What should be possible, but
-difficult? What should be impossible or very difficult?_
+
+It should be easy to find out about classes. It should be easy to register for
+classes, it should be easy to get more information about classes.
+
+It should not be Turing complete. It should be hard or impossible to interface
+with non-registration related things.
```

Comment:
Agreed. Having worked with people who try to do way too much in SQL, it's horrid.